src/styles.scss

```diff
@@ -164,6 +164,13 @@ footer.docs {
   width: calc(100% - 280px);
 }
 
+blockquote.warning {
+  border-left: 3px solid rgb(228, 167, 2);
+  padding-left: 12px;
```

Comment:
This is currently how it looks:
![image](https://user-images.githubusercontent.com/206116/203646346-cbb78348-13ca-4512-820d-7d6944290cbf.png)

Changing to
```suggestion
  padding: 15px;
```
yields

![image](https://user-images.githubusercontent.com/206116/203646523-cceafb48-f3fa-4700-b674-b5eba6dd6e63.png)


The left indentation is because it's a `blockquote`. I suggest we change it to `<div>` and add a vertical margin

```
margin: 15px 0;
```
to yield

![image](https://user-images.githubusercontent.com/206116/203646776-e8ad6571-a0c9-4ff5-ae3b-062d21943b9f.png)

Reply:
Nice! Thanks for playing with the numbers! [4d5ffcf](https://github.com/gazebo-web/gazebosim-web-frontend/pull/12/commits/4d5ffcf6ee958f21915690a87c81895f8df8a31c)